index.html

```diff
@@ -232,111 +232,111 @@ <h1>Connect with Fossasia on Social Media</h1>
-                   <a href="img/PastMemories/10.jpg" data-lightbox="Gallery">
-                       <img id="memory10" class="memories_img" src="img/PastMemories/10.jpg" >
-                   </a>
-              </div>
-              <div class="col-md-2 spacing-0">
-                   <a href="img/PastMemories/11.jpg" data-lightbox="Gallery">
-                       <img id="memory11" class="memories_img" src="img/PastMemories/11.jpg" >
-                   </a>
-              </div>
-              <div class="col-md-2 spacing-0">
-                   <a href="img/PastMemories/12.jpg" data-lightbox="Gallery">
-                       <img id="memory12" class="memories_img" src="img/PastMemories/12.jpg" >
-                   </a>
-              </div>
-         </div>
-         <div class="row">
-              <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
-                   <a href="img/PastMemories/13.jpg" data-lightbox="Gallery">
-                       <img id="memory13" class="memories_img" src="img/PastMemories/13.jpg" >
-                   </a>
-              </div> 
-              <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
-                   <a href="img/PastMemories/14.jpg" data-lightbox="Gallery">
-                      <img id="memory14" class="memories_img" src="img/PastMemories/14.jpg" >
-                   </a>
-              </div>
-              <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
-                   <a href="img/PastMemories/15.jpg" data-lightbox="Gallery">
-                      <img id="memory15" class="memories_img" src="img/PastMemories/15.jpg" >
-                   </a>
-              </div>
-              <div class="col-sm-3 col-md-2 spacing-0">
-                   <a href="img/PastMemories/16.jpg" data-lightbox="Gallery">
-                      <img id="memory16" class="memories_img" src="img/PastMemories/16.jpg" >
-                   </a>
-              </div>
-              <div class="col-md-2 spacing-0">
-                   <a href="img/PastMemories/17.jpg" data-lightbox="Gallery">
-                      <img id="memory17" class="memories_img" src="img/PastMemories/17.jpg" >
-                   </a>
-              </div>
-              <div class="col-md-2 spacing-0">
-                   <a href="img/PastMemories/18.jpg" data-lightbox="Gallery">
-                      <img id="memory18" class="memories_img" src="img/PastMemories/18.jpg" >
-                   </a>
-              </div>
-        </div>
-        <div class="align-middle" id="memories_div" >
-           <i class="glyphicon glyphicon-camera" id="memories_icon" ></i><br>
-           <h1 id="memories_heading" >Fossasia Album</h1><br>
-           <h3 id="memories_link">Follow our photostream on
-              <a id="flickr" href="https://www.flickr.com/photos/fossasia/" target="_blank">flickr</a>
-           </h3>        
-        </div>
+  <div class="container-fluid" id="memories">
+    <div class="row">
```

Comment:
Please don't change the intendation.

Reply:
I didn't..  it is done automaticly by prettier plugin